echo/types.go

```diff
@@ -0,0 +1,6 @@
+package echo
+
+type Advisory map[string]map[string]struct {
+	Severity     string `json:"severity,omitempty"`
```

Comment:
Hi @orizerah
I'm checking the DB from https://github.com/aquasecurity/trivy-db/pull/528#issuecomment-2898029577

I found that https://advisory.echohq.com/data.json does not have severity levels.
Why did you add this field?
Does `echo` plan to add severity levels?